frontend/class-twitter.php

```diff
@@ -594,6 +598,31 @@ private function image_thumbnail_output( $post_id = 0 ) {
 		return false;
 	}
 
+	/**
+	 * Retrieve the image from the content.
+	 *
+	 * @param int $post_id The post id to extract the images from.
+	 *
+	 * @return bool True when images output succeeded.
+	 */
+	private function image_from_content_output( $post_id ) {
```

Comment:
This seems duplicate. Perhaps it's a good idea to create a separate issue to abstract this.